### PR TITLE
Update npm scripts

### PR DESCRIPTION
**Pull request by garatortiz**
This PR updates and improves the npm scripts

### CODE REVIEW

**Comment by igorbarinov** [comment]
@matiasgaratortiz please follow PR policy for future PRs https://github.com/poanetwork/RFC/issues/3

File: package.json [hunk 1]
```diff
@@ -4,6 +4,12 @@
   "description": "Smart contract and DApp by poa.network to create proof of a physical address",
   "main": "server.js",
   "scripts": {
+    "build": "cd web-dapp && npm run build",
+    "deploy": "cd utils && node deploy_contract",
```
Comment (phahulin) [inline package.json]: Could we rename it to `deploy-on-test`?
`deploy` looks too "prod-like", however this script is only for local tests

File: package.json [hunk 2]
```diff
@@ -4,6 +4,12 @@
...
+    "postinstall": "cd web-dapp && npm install",
+    "prestart": "npm run build",
+    "start": "cd web-dapp && node server",
+    "start-rpc": "./scripts/start_rpc.sh",
```
Comment (phahulin) [inline package.json]: And this one - rename to `start-testrpc`.

File: README.md [hunk 3]
```diff
@@ -40,33 +37,31 @@ module.exports = function (cfg_public) {
 ```
 If this file is present, its keys will add to/replace keys in `web-dapp/server-config.js`.
 
-5. open new tab in your terminal, cd to `utils` folder and start testrpc with a set of predefined acounts
+5. open new tab in your terminal, and start testrpc with a set of predefined acounts
 ```
-cd utils
-bash start_rpc.sh
+npm run start-rpc
```
Comment (phahulin) [inline README.md]: Change here to `npm run start-testrpc` (see below)

File: README.md [hunk 4]
```diff
@@ -40,33 +37,31 @@ module.exports = function (cfg_public) {
...
 ```
 leave this tab opened until your test is complete.
 
-6. in the first tab of your terminal, go to `utils` folder too and deploy the contract
+6. in the first tab of your terminal deploy the contract
 ```
-cd utils
-node deploy_contract
+npm run deploy
```
Comment (phahulin) [inline README.md:48]: Change here to `npm run deploy-on-test` (see below).

**Pull request merged**